components/max.scss

```diff
@@ -12,6 +12,7 @@
 @import url("./image/image.css");
 @import url("./input/input.css");
 @import url("./text/text.css");
+@import url("./slider/slider.css");
 
 :root {
   --primary: #129bfd;
```

Comment:
If this is a primary color you can name it like --color-primary. It will make more sense to others as well.